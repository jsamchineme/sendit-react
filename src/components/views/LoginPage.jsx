import React from 'react';
import { NavLink } from 'react-router-dom';

const LoginPage = () => {
  return (
    <div className="wrapper">
      <section className="page-content">
        <section className="mast no-bg paddless">
          <a href="index.html" className="logo-text-group">
            <div className="logo"><img src="assets/img/sendit-pickup-ondemand.svg" alt="sendit-logo" /></div>
            <div className="text">Send<span>IT</span></div>
          </a>
        </section>
        <div className="login-box">
          <form action="">
            <div className="form-header">
              Login
            </div>
            <div className="form-body">
              <div className="input-group transparent-box">
                <label htmlFor="email">Email</label>   
                <input type="email" placeholder="you@email.com" id="email" required />
              </div>
              <div className="input-group transparent-box">
                <label htmlFor="password">Password</label>
                <input type="password" placeholder="your password" id="password" required /> 
              </div>
              <div className="input-footer" />
              <div className="row text-center">
                <div className="column">
                  <button className="btn submit">Login</button>
                </div>
                <div className="actions column text-center">
                  <div className="v-gap-2" />
                  <div className>
                    <NavLink to="/signup">Sign up</NavLink> 
                    <span style={{marginRight: 10, display: 'inline-block'}} />
                    <NavLink to="/forgot-password">Forgot password</NavLink>
                    <span style={{marginRight: 10, display: 'inline-block'}} />
                    <NavLink to="/admin-login">Admin Login</NavLink>
                  </div>
                </div>
              </div>
            </div>
          </form>
        </div>
      </section>
      <div className="page-background">
        <div className="overlay" />
      </div>
    </div>

  );
}

export default LoginPage;