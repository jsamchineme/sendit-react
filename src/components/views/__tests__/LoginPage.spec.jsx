import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../LoginPage';

describe('Login Page', () => {
  it('should render correctly', () => {
    const component = shallow(<LoginPage />);

    expect(component).toMatchSnapshot();
  });
});