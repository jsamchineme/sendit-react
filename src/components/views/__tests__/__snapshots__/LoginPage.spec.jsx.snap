// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should render correctly 1`] = `
<div
  className="wrapper"
>
  <section
    className="page-content"
  >
    <section
      className="mast no-bg paddless"
    >
      <a
        className="logo-text-group"
        href="index.html"
      >
        <div
          className="logo"
        >
          <img
            alt="sendit-logo"
            src="assets/img/sendit-pickup-ondemand.svg"
          />
        </div>
        <div
          className="text"
        >
          Send
          <span>
            IT
          </span>
        </div>
      </a>
    </section>
    <div
      className="login-box"
    >
      <form
        action=""
      >
        <div
          className="form-header"
        >
          Login
        </div>
        <div
          className="form-body"
        >
          <div
            className="input-group transparent-box"
          >
            <label
              htmlFor="email"
            >
              Email
            </label>
            <input
              id="email"
              placeholder="you@email.com"
              required={true}
              type="email"
            />
          </div>
          <div
            className="input-group transparent-box"
          >
            <label
              htmlFor="password"
            >
              Password
            </label>
            <input
              id="password"
              placeholder="your password"
              required={true}
              type="password"
            />
          </div>
          <div
            className="input-footer"
          />
          <div
            className="row text-center"
          >
            <div
              className="column"
            >
              <button
                className="btn submit"
              >
                Login
              </button>
            </div>
            <div
              className="actions column text-center"
            >
              <div
                className="v-gap-2"
              />
              <div
                className={true}
              >
                <NavLink
                  to="/signup"
                >
                  Sign up
                </NavLink>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "marginRight": 10,
                    }
                  }
                />
                <NavLink
                  to="/forgot-password"
                >
                  Forgot password
                </NavLink>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "marginRight": 10,
                    }
                  }
                />
                <NavLink
                  to="/admin-login"
                >
                  Admin Login
                </NavLink>
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </section>
  <div
    className="page-background"
  >
    <div
      className="overlay"
    />
  </div>
</div>
`;
